Assets/Scripts/Player/Animation.cs

```diff
@@ -0,0 +1,63 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Animation : MonoBehaviour
+{
+    public float movementSpeed = 3f;
+    public float sprintingSpeed = 6f;
+    public float superSpeed = 20f;
+    private float actuallSpeed;
+    public Vector2 movement;
+    public Rigidbody2D rigidbody;
+
+
+    public Animator playerAnimator;
+    public float hf = 0.0f;
+    public float vf = 0.0f;
```

Comment:
a) What even does this store? I still have no clue.
b) Is there a reason why this needs to be an instance variable?